capture/fswebcam.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+
+import capture
+from utils.log import debug
+
+
+class Fswebcam(capture.ImgCapture):
```

Comment:
How about using class names that reflect more explicitly the class hierarchy, e.g. `BaseCaptureDevice`, `FswebcamCaptureDevice`, `MockCaptureDevice` ?